blog/2024-09-16-ratify-1.3.0.mdx

```diff
@@ -0,0 +1,66 @@
+---
+slug: ratify-v1.3.0-is-available
+title: Ratify v1.3.0 is available! 
+authors: [ratify]
+tags: [verification]
+---
+
+We are excited to announce the release of Ratify v1.3.0!
+
+Ratify is now a CNCF sandbox project! We sincerely thank the exceptional community for invaluable contributions and unwavering support in reaching this significant milestone.
+Ratify is a verification framework that ensures container images and supply chain artifacts stored in a registry are thoroughly vetted and trustworthy. 
+By [integrating Ratify into your Kubernetes](https://notaryproject.dev/blog/2023/announcing-major-release/#integration-with-admission-controller-for-kubernetes-usage) workflow, 
+you can safeguard your application deployment against potential threats and vulnerabilities, thereby greatly enhancing the security of your applications.
+
+Read further to learn about the exciting new features and how they can be beneficial to you!
+
+## Features and Deprecations
+
+Key features in this release include:
+- Support of validating Notary Project signatures with timestamping
+- Support of periodic retrieval of keys and certificates stored in a KMS (Key Management System)
+- Improving error messages for artifact verification
+- Introducing new configuration for Cosign keyless verification
+
+### Support of validating Notary Project signatures with timestamping
+
+Ratify now supports verifying Notary Project signatures with timestamping. 
+Since [Notation v1.2.0 release](https://notaryproject.dev/blog/2024/announcing-notation-v1-2/), Notary Project supports [RFC 3161](https://www.rfc-editor.org/rfc/rfc3161)-compliant timestamping. 
+This enhancement extends the trust of signatures created within the certificate's validity period by trusting on a Timestamping Authority (TSA), 
+enabling successful signature verification even after the certificates have expired. 
+Timestamping reduces costs by eliminating the need to periodically re-sign images due to certificate expiry, which is critical when using short-lived certificates. 
+To validate Notary Project signatures that include timestamps from a TSA, it's essential for a verifier to determine trusted entities both as the image signer and as the TSA. In Ratify, configure trust stores and trust policies in the Notation verifier to establish this trust. For more information, see the [configuration of Notation verifier](https://ratify.dev/docs/next/plugins/verifier/notation/#configuration).

```

Comment:
In Ratify, trust stores and trust policies can be configured in the Notation verifier to establish this trust.